os2forms_failed_jobs.install

```diff
@@ -90,3 +99,10 @@ function os2forms_failed_jobs_update_9001(): void {
 
   $schema->createTable($table_name, $table_schema);
 }
+
+/**
+ * Add existing queue elements to new table.
+ */
+function os2forms_failed_jobs_update_9002(): void {
+  \Drupal::getContainer()->get(Helper::class)->handleImport();
```

Comment:
Use `Drupal::service(Helper::class)` to get the service as we do it throughout the OS2Forms project.

Reply:
Fixed